A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Elec.js

```diff
@@ -0,0 +1,1081 @@
+            this.values.EMERGEN_VOLTS = this.getSimVar_Number('EMER_VOLTAGE')
+            this.values.EMERGEN_AMPS = this.getSimVar_Number('EMER_AMPERAGE')
+            this.values.EMERGEN_FREQ = this.getSimVar_Number('EMER_FREQ')
+
+            this.buses.AC1 = this.getSimVar_BusEnum('AC_BUS1')
+            this.buses.AC2 = this.getSimVar_BusEnum('AC_BUS2')
+            this.buses.ACESS = this.getSimVar_BusEnum('AC_ESS')
+            this.buses.ACSHED = this.getSimVar_BusEnum('AC_SHED')
+            this.buses.GALLEY_SHED = this.getSimVar_BusEnum('GALLEY_SHED')
+            this.buses.DC1 = this.getSimVar_BusEnum('DC_BUS1')
+            this.buses.DC2 = this.getSimVar_BusEnum('DC_BUS2')
+            this.buses.DCBAT = this.getSimVar_BusEnum('DC_BATBUS')
+            this.buses.DCESS = this.getSimVar_BusEnum('DC_ESS')
+            this.buses.DCSHED = this.getSimVar_BusEnum('DC_SHED')
+            this.buses.HOT1 = this.getSimVar_BusEnum('HOT_BUS1')
+            this.buses.HOT2 = this.getSimVar_BusEnum('HOT_BUS2')
+
+            this.systems.TR1 = this.getSimVar_ONOFF('TR1_ONLINE')
+            this.systems.TR2 = this.getSimVar_ONOFF('TR2_ONLINE')
+            this.systems.TRESS = this.getSimVar_ONOFF('TRESS_ONLINE')
+            this.values.TR1_VOLTS = this.getSimVar_Number('TR1_VOLTAGE')
+            this.values.TR2_VOLTS = this.getSimVar_Number('TR2_VOLTAGE')
+            this.values.TRESS_VOLTS = this.getSimVar_Number('TRESS_VOLTAGE')
+            this.values.TR1_AMPS = this.getSimVar_Number('TR1_AMPERAGE')
+            this.values.TR2_AMPS = this.getSimVar_Number('TR2_AMPERAGE')
+            this.values.TRESS_AMPS = this.getSimVar_Number('TRESS_AMPERAGE')
+
+            this.systems.STATINV = this.getSimVar_ONOFF('STATICINV_ONLINE')
+            this.values.STATINV_VOLTS = this.getSimVar_Number('STATICINV_VOLTAGE')
+            this.values.STATINV_AMPS = this.getSimVar_Number('STATICINV_AMPERAGE')
+            this.values.STATINV_FREQ = this.getSimVar_Number('STATICINV_FREQ')
+
+            this.internals.AC_AVAIL = this.getSimVar_Bool('ACPowerAvailable')
+            this.internals.DC_AVAIL = this.getSimVar_Bool('DCPowerAvailable')
+
+            this.switches.ACESS_FEED_AUTO = this.getSimVar_Bool('A32NX_ELEC_ACESSFEED_TOGGLE')
+            this.switches.COMMERCIAL_ON = this.getSimVar_Bool('A32NX_ELEC_COMMERCIAL_TOGGLE')
+            this.switches.GALYCAB_ON = this.getSimVar_Bool('A32NX_ELEC_GALYCAB_TOGGLE')
+        }
+
+        getSimVar_Bool(varName) {
+            return Math.random() > 0.5
+        }
+
+        getSimVar_Number(varName) {
+            return (Math.random() * 300).toFixed(0)
+        }
+
+        getSimVar_ONOFF(varName) {
+            return Math.random() > 0.5 ? 'ON' : 'OFF'
+        }
+
+        getSimVar_BusEnum(varName) {
+            const index = Math.floor(Math.random() * 11)
+            return ENUM_BUS[index]
+        }
+
+        systemDraw() {
```

Comment:
Seems like here we update the DOM every single frame. This can have a pretty big impact on performance. No need to change anything for now as it's a pretty blurry topic rn, but we should think about how to cache and diff state in the future.